optaplanner-core/src/main/java/org/optaplanner/core/impl/domain/common/ReflectionHelper.java

```diff
@@ -114,7 +114,7 @@ public static Method getGetterMethod(Class containingClass, String propertyName)
                 + (propertyName.isEmpty() ? "" : propertyName.substring(0, 1).toUpperCase() + propertyName.substring(1));
         try {
             return containingClass.getMethod(getterName);
-        } catch (NoSuchMethodException e) {}
+        } catch (NoSuchMethodException e) { }
```

Comment:
Because the WhitespaceAround (quite a useful check) can't be disabled to allow empty blocks. Fixed to
```
...
} catch (NoSuchMethodException e) {
    // intentionally empty
}
...
```